tangelo/problem_decomposition/dmet/dmet_problem_decomposition.py

```diff
@@ -569,6 +601,86 @@ def _compute_energy(self, mf_frag, onerdm, twordm, fock_frag_copy, t_list, onein
+                         + np.einsum("ij,ij->", fock_frag_copy, onerdm_b ) \
+                         + 0.5 * np.einsum("ijkl,ijkl->", twoint, twordm_aa ) \
+                         + 0.5 * np.einsum("ijkl,ijkl->", twoint, twordm_ab ) \
+                         + 0.5 * np.einsum("ijkl,klij->", twoint, twordm_ab ) \
+                         + 0.5 * np.einsum("ijkl,ijkl->", twoint, twordm_bb )
+
+        # Calculate fragment expectation value
+        fragment_energy_one_rdm = 0.25 * np.einsum("ij,ij->", onerdm_a[: norb, :], fock[: norb, :] + oneint[: norb, :]) \
+                               + 0.25 * np.einsum("ij,ij->", onerdm_a[:, : norb ], fock[:, : norb] + oneint[:, : norb]) \
+                               + 0.25 * np.einsum("ij,ij->", onerdm_b[: norb, :], fock[: norb, :] + oneint[: norb, :]) \
+                               + 0.25 * np.einsum("ij,ij->", onerdm_b[:, : norb ], fock[:, : norb] + oneint[:, : norb])
+
+        fragment_energy_twordm = 0.125 * np.einsum("ijkl,ijkl->", twordm_aa[: norb, :, :, :], twoint[: norb, :, :, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_aa[:, : norb, :, :], twoint[:, : norb, :, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_aa[:, :, : norb, :], twoint[:, :, : norb, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_aa[:, :, :, : norb], twoint[:, :, :, : norb]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_ab[: norb, :, :, :], twoint[: norb, :, :, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_ab[:, : norb, :, :], twoint[:, : norb, :, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_ab[:, :, : norb, :], twoint[:, :, : norb, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_ab[:, :, :, : norb], twoint[:, :, :, : norb]) \
+                               + 0.125 * np.einsum("klij,ijkl->", twordm_ab[:, :, : norb, :], twoint[: norb, :, :, :]) \
+                               + 0.125 * np.einsum("klij,ijkl->", twordm_ab[:, :, :, : norb], twoint[:, : norb, :, :]) \
+                               + 0.125 * np.einsum("klij,ijkl->", twordm_ab[: norb, :, :, :], twoint[:, :, : norb, :]) \
+                               + 0.125 * np.einsum("klij,ijkl->", twordm_ab[:, : norb, :, :], twoint[:, :, :, : norb]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_bb[: norb, :, :, :], twoint[: norb, :, :, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_bb[:, : norb, :, :], twoint[:, : norb, :, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_bb[:, :, : norb, :], twoint[:, :, : norb, :]) \
+                               + 0.125 * np.einsum("ijkl,ijkl->", twordm_bb[:, :, :, : norb], twoint[:, :, :, : norb])
+
+        fragment_energy = fragment_energy_one_rdm + fragment_energy_twordm
```

Comment:
variable name style inconsistent (one_rdm, twordm).
If you want to change / shorten any variable names in this file, please go ahead (use search and replace)